### PR TITLE
Update images usage to contain positional arguments

### PR DESCRIPTION
**Pull request by davidhsingyuchen**
PR updates the `images` usage line to be consistent with that in [README](https://github.com/containerd/nerdctl#whale-blue_square-nerdctl-images). Changed `OPTIONS` to `flags` because the latter is the term that's displayed in `--help`, and it's also used in other commands like `start`, `stop`, etc.

Before PR:

```
Usage: nerdctl images [flags]
```

After PR:

```
Usage: nerdctl images [flags] [REPOSITORY[:TAG]]
```

Signed-off-by: Hsing-Yu (David) Chen <davidhsingyuchen@gmail.com>

### CODE REVIEW

File: cmd/nerdctl/images.go [hunk 1]
```diff
@@ -59,14 +59,15 @@ Properties:
...
-		SilenceErrors:     true,
+		Use:                   "images [flags] [REPOSITORY[:TAG]]",
+		Short:                 shortHelp,
+		Long:                  longHelp,
+		Args:                  cobra.MaximumNArgs(1),
+		RunE:                  imagesAction,
+		ValidArgsFunction:     imagesShellComplete,
+		SilenceUsage:          true,
+		SilenceErrors:         true,
+		DisableFlagsInUseLine: true,
```
Comment (manugupt1) [inline cmd/nerdctl/images.go:70]: Q: Why was this added?

**Review by manugupt1 — APPROVED** [review]

**Review by yardenshoham — APPROVED** [review]

**Review by AkihiroSuda — APPROVED** [review]
Thanks
